include/platform/tasklet.h

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
In general, APIs in `include/platform` are platform abstractions backed by platform-specific implementations.  Are you looking to develop platform-specific implementations of these tasklet APIs?